agent/installer/installer.go

```diff
@@ -52,14 +52,18 @@ func getSupportedRegistry(ob algo.OutputBuilder) registry {
 
 		// BYOH Bundle Repository. Associate bundle with installer
 		linuxDistro := "Ubuntu_20.04.1_x86-64"
+		addBundleInstaller(linuxDistro, "v1.21.*", &algo.Ubuntu20_4K8s1_22{})
 		addBundleInstaller(linuxDistro, "v1.22.*", &algo.Ubuntu20_4K8s1_22{})
+		addBundleInstaller(linuxDistro, "v1.23.*", &algo.Ubuntu20_4K8s1_22{})
 
 		/*
 		 * PLACEHOLDER - ADD MORE K8S VERSIONS HERE
 		 */
 
 		// Match any patch version of the specified Major & Minor K8s version
+		reg.AddK8sFilter("v1.21.*")
 		reg.AddK8sFilter("v1.22.*")
+		reg.AddK8sFilter("v1.23.*")
```

Comment:
unit tests please? :)

Reply:
We have unit tests for the filters from my previous PR. If you have something more specific  in mind please let me know and I will code it.

Reply:
The current tests are for `v1.22.*` minor series. Ideally there should be 2 kinds of tests. One specifically for minor series support - something like if K8sFilter is set to`v1.foo.*`, it should not error on installing `v1.foo.bar`.

This PR specifically talks about adding support for a concrete minor version - `21.*` and `23.*` (without you having to add the filter yourself, the registry should accommodate these series)

Thoughts? The reason being there are no backing tests for the code / filters that you introduced. If I were to accidentally remove `reg.AddK8sFilter("v1.21.*")`, there would be no signal screaming that this is not allowed.

Reply:
This might involve structuring the current tests a bit differently. If you want to take this up in a follow-up PR, I'm good with it as we don't want to block @dharmjit :)

Reply:
Okay, let's not block @dharmjit  yet however I will start working on a new PR that is implementing the tests right away. 
Thanks!

Reply:
Cool, please open a corresponding issue to track this effort.